website/docs/r/nginx_configuration.html.markdown

```diff
@@ -10,6 +10,8 @@ description: |-
 
 Manages the configuration for a Nginx Deployment.
 
+~> **NOTE** Do not use the independent `azurerm_nginx_configuration` resource anymore due to a [known API issue](https://docs.nginx.com/nginxaas/azure/known-issues/#i-classfa-solid-fa-bug-stylecolore4002bi-terraform-shows-an-error-while-trying-to-manage-configuration-of-a-fresh-deployment-id-891). Use the `configuration` block of [azurerm_nginx_deployment](./nginx_deployment#configuration) instead.
```

Comment:
should we just remove this resource documentation entirely as it no longer works on creation & deprecate the resouce?